web/src/scripts/containers/ProjectNavigator.jsx

```diff
@@ -104,17 +108,20 @@ class ProjectNavigator extends Component {
 
   handlePreviewClick = async (node) => {
     const {rootPath, textEditorCompositeActions} = this.props
+    this.setState({currentPreview: node.path})
```

Comment:
I think this could go in another PR, but really the play button is determined by the entry point, which we pull out of a file. So wherever we have that in the store, we should pass that through to here and there should be a redux action to set it